webpackjs/components/About/Body.js

```diff
@@ -1,29 +1,62 @@
 import React from "react";
 
 const Body = ({}) => (
-  <div className="row px-3 px-sm-0">
-    <h1>Data & Technology Services</h1>
-    <h2>
-      We use technology to solve real problems for real people related to
-      mobility in Austin
-    </h2>
+  <div className="px-3 px-sm-0">
```

Comment:
@amenity, the `row` that I removed here was the main problem with titles bunching together. The rest was just for better spacing and hierarchy